### ISSUE TITLE
[FR] Disable Dual X unused steppers including extruder

### ISSUE THREAD

**Issue by bruce356** [MarlinFirmware/Marlin#8045]
Hi, on a dual X carriage Cartesian printer is it possible to disable the carriage that is not being used. Very often only one extruder is being used to print with (X or X2) so the other carriage is not required to be powered up for the duration of that print. How if it is possible does one disable any unused stepper motors including the unused extruder stepper motor. If its currently not possible can this option be added.
Thanks and regards.

**Comment by thinkyhead** [comment]
Yes, this would be a good idea to save wasted energy. The X/X2 and E0/E1 steppers should be turned on/off and timed-out separately when not in duplication mode.

**Comment by boelle** [comment]
wrong label

**Comment by InsanityAutomation** [comment]
Id consider this as needing another mode in M605 that will force the unused head to rehome on tool change otherwise there will be issues. Unused extruder maybe but not the movement stepper.

Regardless this is something id be very hesitant to use or recommend using.
